web/templates/nav.html

```diff
@@ -17,14 +17,14 @@
 
 	<div class="collapse navbar-collapse" id="navbarSupportedContent">
 		<ul class="navbar-nav align-items-center">
-			<li class="nav-item active ml-md-3">
-				<a class="nav-link" href="#">
-					Home<span class="sr-only">(current)</span>
+			<li class="nav-item ml-md-3">
```

Comment:
Ideally, we find a way to keep the active tag. Maybe we inject it into the nav.html template.

Reply:
Yeah, I guess it does help. Added it back by adding:
`home_tag=' active', about_tag=''` in the home render, and `home_tag='', about_tag=' active'` in the about render, then adding `{{ home_tag }}` and `{{ about_tag }}` into their respective places on the nav.html template.

I don't love the solution, but it gets us there.